sdk/daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/RunnerMainConfig.scala

```diff
@@ -287,15 +284,6 @@ private[script] object RunnerMainConfigIntermediate {
         s"Uploads the dar before running. Only available over GRPC. Default behaviour is to upload with --all, not with --script-name."
       )
 
-    opt[Unit]("enable-contract-upgrading")
```

Comment:
We may have to update the doc.

Reply:
where it is ?

Reply:
[here](https://github.com/digital-asset/docs.daml.com/blob/d46ff4ae96dedf8c3f91aaaf76ecb3c4adc72be5/docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst?plain=1#L605) and below in the same document.